r-package/grf/R/instrumental_forest.R

```diff
@@ -25,14 +25,15 @@
 #'                       If NULL, each observation receives equal weight. Default is NULL.
 #' @param clusters Vector of integers or factors specifying which cluster each observation corresponds to.
 #'                 Default is NULL (ignored).
-#' @param samples.per.cluster If sampling by cluster, the number of observations to be sampled from
-#'                            each cluster when training a tree. If NULL, we set samples.per.cluster to the size
-#'                            of the smallest cluster. If some clusters are smaller than samples.per.cluster,
-#'                            the whole cluster is used every time the cluster is drawn. Note that
-#'                            clusters with less than samples.per.cluster observations get relatively
-#'                            smaller weight than others in training the forest, i.e., the contribution
-#'                            of a given cluster to the final forest scales with the minimum of
-#'                            the number of observations in the cluster and samples.per.cluster. Default is NULL.
+#' @param equalize.cluster.weights If FALSE, each unit is given the same weight (so that bigger
+#'  clusters get more weight). If TRUE, each cluster is given equal weight in the forest. In this case,
```

Comment:
The indentation here (and in other similar spots) is not consistent.

Reply:
I thought it was easier to read when the text was not indented all the way to the right (like in for example honesty.prune.leaves, they also do it [here](https://github.com/tidyverse/dplyr/blob/master/R/group-by.r#L30)). If you agree reading from the left is easier we can harmonize the docstring to this format in another PR?

Reply:
I agree that we should stick with our current indentation scheme for now, so we don't have inconsistencies within the same method comment. I also think that the indentation scheme you propose is more readable, we could use it for new methods and start to switch existing ones over.